index.html

```diff
@@ -37,6 +37,6 @@
 <canvas id="mainCanvas"></canvas>
 
 <script src="script.js"></script>
-<script src="pencil.js"></script>
+<script src="js\pencil.js"></script>
```

Comment:
Э-э-э. Обратный слеш. Он точно хорошо работает?

Reply:
Работает, но по общей договоренности будем использовать "/"
```suggestion
<script src="js/pencil.js"></script>
```